NewHorizons/Builder/Body/WaterBuilder.cs

```diff
@@ -47,10 +47,13 @@ public static RadialFluidVolume Make(GameObject planetGO, Sector sector, OWRigid
 
             GameObject waterGO = new GameObject("Water");
             waterGO.SetActive(false);
-            waterGO.layer = 15;
             waterGO.transform.parent = sector?.transform ?? planetGO.transform;
             waterGO.transform.localScale = new Vector3(waterSize, waterSize, waterSize);
 
+            // Don't ignore sun when not under clouds
+            waterGO.layer = 0;
+            Delay.FireOnNextUpdate(() => { if (planetGO.FindChild("Sector/SunOverride") != null) waterGO.layer = 15; });
```

Comment:
Do we have something listing the layers with their human readable names instead of writing 15

Reply:
Nvm I dont care enough